rpi/src/platform_rpi.h

```diff
@@ -3,3 +3,24 @@
 #include "platform.h"
 
 void processNetworkQueue();
+
+#define NUM_WORKERS 3
+
+class RPiPlatform : public Platform {
```

Comment:
Since RPi and Linux platforms are very similar, probably make sense for RPiPlatform to inherit from LinuxPlatform and override relevant bits from there. Will result in more code reuse.